Marlin/temperature.cpp

```diff
@@ -588,8 +588,8 @@ void manage_heater()
         disable_heater();
         if(IsStopped() == false) {
           SERIAL_ERROR_START;
-          SERIAL_ERRORLNPGM("Extruder switched off. Temperature difference between temp sensors is too high !");
-          LCD_ALERTMESSAGEPGM("Err: REDUNDANT TEMP ERROR");
+          SERIAL_ERRORLNPGM(MSG_ERR_REDUNDANT_TEMP);
+          LCD_ALERTMESSAGEPGM(REDUNDANT_TEMP);
```

Comment:
The names don't match the defines. As I understand it, any messages bound for the Serial Output are supposed to be in `language.h` – in the untranslated strings section. The theory being that controller software might want to do string matching to deal with some messages. We should find out more specific details about that, because it seems like there  could be standard error number codes, but the accompanying messages should be translatable.